external_search.py

```diff
@@ -383,9 +383,9 @@ def _f(s):
 
         clauses = []
         if languages:
-            clauses.append(dict(terms=dict(language=languages)))
+            clauses.append(dict(terms=dict(language=list(languages))))
```

Comment:
This is the code that uses a set for exclude_languages but a list for languages:
https://github.com/NYPL-Simplified/circulation/blob/master/api/lanes.py#L56